editor/audio_stream_preview.cpp

```diff
@@ -234,6 +246,12 @@ void AudioStreamPreviewGenerator::_notification(int p_what) {
 			to_erase.pop_front();
 		}
 	}
+	if (p_what == NOTIFICATION_ENTER_TREE) {
```

Comment:
These should probably be `else if`, or even better a `switch() { case ... }` statement to be more consistent with the rest of the codebase.

Reply:
hmmm, from what I've seen which is admittedly not much the godot codebase overwhelmingly uses individual `if` statements. I can find a few `else if` usages for checking `p_what` but not many. I've certainly never seen one without explicitly searching for it.

Reply:
From a quick search in vs code it looks like about half of them use `switch`. I thought I read somewhere recently that one of the contributors said `switch was preferred`, I'll see if I can find it 🙂

Reply:
Ok. I'm personally not a fan of C-style switch statements because they just feel syntactic sugar on top of goto's. I know all control structures are syntactic sugar on top of goto's but with switch statements it's especially bad :) I'll definitely just use whatever's convention here though.

Reply:
Both styles are fine. We tend to move towards switches for `_notification` specifically, but here existing code was using `if` and refactoring is not necessarily something that should be done in this PR.

It's fine to do it if you want though.

`else if` on the other hand would be weird, that's not a style we use for `_notification`.